### DISCUSSION TITLE
Can't cmd lua-language-server

### DISCUSSION THREAD

**Discussion by fengtianru117** [williamboman/nvim-lsp-installer#440] [Q&A] [answered]
My setup 
![image](https://user-images.githubusercontent.com/27802344/150670970-0ef74998-55ee-46af-9339-86c3a5e0025d.png)
But i got this
![image](https://user-images.githubusercontent.com/27802344/150670981-b74a552f-4a3c-4ff1-acdb-d3da8384b4e5.png)

I'm sure the `~/.local/share/nvim/lsp_servers/sumneko_lua/extension/server/bin/lua-language-server` is executable
![image](https://user-images.githubusercontent.com/27802344/150671014-06740e2c-f5ce-4292-bba6-b29c373e4502.png)

Here is lsp.log show
![image](https://user-images.githubusercontent.com/27802344/150671025-0aaafffe-b59a-420a-bc6a-bf2336a895d8.png)

How can i solve this?

**Comment by williamboman** [comment, marked as answer]
Hello! Please refer to the Setup section in the README: https://github.com/williamboman/nvim-lsp-installer#setup for the recommended setup. By providing the path manually like you do, it'll be prone for breakage.

As for why your snippet fails is because the `~` is not a real path, but has to be expanded. The shell does this automatically for example. Try wrapping that string with `vim.fn.expand()`.
